src/importing/text-files.md

```diff
@@ -155,13 +151,6 @@ so if you have a file named apple.jpg, the front would say 'apple') and
 the images or audio on the back. If you would like a different
 arrangement of media and filenames, you can [change the note type](../browsing.md) of the created cards afterwards.
 
-## Adding Tags
-
-If you want to add 'tag1' and 'tag2' to every line you’re importing, add
-the following to the top of the text file:
-
-    tags:tag1 tag2
-
```

Comment:
This is distinct from the tags: entry below - note how the first line doesn't need a comment. But I'd prefer we push people to use the more consistent approach mentioned below, so I think it's ok to remove this.

Reply:
Yeah, that's what I thought too.